src/main/java/duck/ToDo.java

```diff
@@ -0,0 +1,29 @@
+package duck;
+
+/**
+ * Represents a simple ToDo task.
+ */
+
+public class ToDo extends Task {
```

Comment:
Should the start of class immediately follow the end of javadoc? 

```suggestion
/**
 * Represents a simple ToDo task.
 */
public class ToDo extends Task {
```